order/src/test/resources/application.properties

```diff
@@ -0,0 +1,7 @@
+#jdbc.driverClassName=org.h2.Driver
```

Comment:
If you don't need application.properties functionality you would delete whole file

Reply:
done